FitGoal/FitGoal.xcodeproj/project.pbxproj

```diff
@@ -76,13 +84,25 @@
 /* End PBXFrameworksBuildPhase section */
 
 /* Begin PBXGroup section */
+		05012C89240D1C940065E560 /* fonts */ = {
+			isa = PBXGroup;
+			children = (
+				05012C9D240D1FCE0065E560 /* Oswald-Medium.ttf */,
+				05012C9F240D1FD70065E560 /* Roboto-Bold.ttf */,
+				05012CA0240D1FD70065E560 /* Roboto-Light.ttf */,
+				05012C9E240D1FD70065E560 /* Roboto-Regular.ttf */,
+			);
+			path = fonts;
+			sourceTree = "<group>";
+		};
 		F142339E240CF77D000E8873 = {
 			isa = PBXGroup;
 			children = (
 				F14233A9240CF77D000E8873 /* FitGoal */,
 				F14233C0240CF77F000E8873 /* FitGoalTests */,
 				F14233CB240CF77F000E8873 /* FitGoalUITests */,
 				F14233A8240CF77D000E8873 /* Products */,
+				05012C89240D1C940065E560 /* fonts */,
```

Comment:
The `fonts` directory should be inside the `FitGoal` directory, not as a top level folder.